packages/calypso-e2e/src/lib/pages/jetpack-forms-inbox-page.ts

```diff
@@ -0,0 +1,84 @@
+import { Page } from 'playwright';
+import { envVariables } from '../..';
+
+/**
+ * Page representing the Jetpack Forms Inbox page. (The "Inbox", non-"Classic" view for forms responses.)
+ */
+export class JetpackFormsInboxPage {
+	private page: Page;
+
+	/**
+	 * Constructs an instance of the component.
+	 *
+	 * @param {Page} page The underlying page.
+	 */
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	/**
+	 * Visit the Jetpack Forms Inbox page.
+	 *
+	 * @param siteUrlWithProtocol
+	 */
+	async visit( siteUrlWithProtocol: string ): Promise< void > {
+		const url = new URL(
+			// The query arg is necessary for sites that were using the "Classic" feedback view.
+			'/wp-admin/admin.php?page=jetpack-forms&dashboard-preferred-view=modern',
+			siteUrlWithProtocol
+		);
+		await this.page.goto( url.href, { timeout: 20 * 1000 } );
+	}
+
+	/**
+	 * Click on a response row that has the provided text.
+	 *
+	 * @param {string} text The text to match in the row. Using the name field is a good choice.
+	 */
+	async clickResponseRowByText( text: string ): Promise< void > {
```

Comment:
**nit:non-blocking** - most methods that only accept one form of parameter to locate the target like this one, we've simply named "clickResponseRow" or similar. The "...ByText" part feels unnecessary to me.